lib/app.rb

```diff
@@ -52,11 +52,11 @@ def run_commands
 
       commands = @display.get_user_commands
       system 'clear'
-      @simulator.evaluate(commands)
-      @simulator.board.print_board
-      puts 'Feel free to enter another set of commands'
+      @simulator.evaluate(commands) { |outcome| @display.print_outcome(@simulator.robot, @simulator.board, outcome) }
+      @display.print_board(@simulator.board.board_with_axis)
+      @display.enter_another_command
```

Comment:
is it `Display`'s responsibility to take user input with `.enter_another_command` ? is this something else? user interface? say you had a joystick connected? or a Wii controller?